src/scripts/main.js

```diff
@@ -1,3 +1,16 @@
 'use strict';
 
-// write your code here
+const populations = [...document.querySelectorAll('.population')].map((el) => {
+  return +el.innerText.replace(/,/g, '');
+});
+const totalPopulationElement = document.querySelector('.total-population');
+const averagePopulationElement = document.querySelector('.average-population');
+const totalPopulation = populations.reduce((a, b) => a + b);
+const averagePopulation = totalPopulation / 10;
+const toFormated = (number) => {
```

Comment:
The function 'toFormated' appears to have a typo. It should be 'toFormatted' to correctly represent its purpose.